packages/transformers/webext/src/schema.js

```diff
@@ -0,0 +1,402 @@
+// @flow strict-local
+import type {SchemaEntity} from '@parcel/utils';
+import {extname} from 'path';
+
+// Non-comprehensive, but should cover the vast majority of users
+// SVGs not supported in WebKit/Blink
+const IMG_EXT = new Set(['jpg', 'jpeg', 'png', 'svg', 'webp', 'gif']);
+
+const validateVersion = (ver: string): ?string => {
+  const parts = ver.split('.', 5);
+  if (parts.length == 5) return 'Extension versions to have at most three dots';
+  // greater than to allow for NaN to fail
+  if (parts.every(part => +part[0] > 0 && +part < 65536)) return;
```

Comment:
I found this bug already:

<img width="610" alt="Screen Shot 2020-11-01 at 18 16 54" src="https://user-images.githubusercontent.com/1024246/97815746-15f11200-1c6f-11eb-81b9-5640406db11f.png">

I think it should be 
```js
if (parts.every(part => +part >= 0 && +part < 65536)) return;
```

Reply:
(Apart from that, please use `Number(part)`)